releases/nightly-build/compose-files/Makefile

```diff
@@ -0,0 +1,91 @@
+.PHONY: portainer portainer-down run run-no-sec run-mqtt run-mqtt-no-sec run-arm run-arm-no-sec run-mqtt-arm run-mqtt-arm-no-sec down clean remove
+
+portainer:
+	docker-compose -p portainer -f docker-compose-portainer.yml up -d
+
+portainer-down:
+	docker-compose -p portainer -f docker-compose-portainer.yml down
+
+run:
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		-f docker-compose-nexus-add-security.yml \
+		up -d
+
+run-no-sec:
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		up -d
+
+run-mqtt:
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		-f docker-compose-nexus-add-mqtt.yml \
+		-f docker-compose-nexus-add-security.yml \
+		up -d
+
+run-mqtt-no-sec:
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		up -d
+
+run-arm:
+	ARCH="-arm64" \
+	REDIS_ARCH="arm64v8/" \
+	KONG_UBUNTU="-ubuntu" \
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		-f docker-compose-nexus-add-security.yml \
+		up -d
+
+run-arm-no-sec:
+	ARCH="-arm64" \
+	REDIS_ARCH="arm64v8/" \
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		up -d
+
+run-mqtt-arm:
+	ARCH="-arm64" \
+	REDIS_ARCH="arm64v8/" \
+	KONG_UBUNTU="-ubuntu" \
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		-f docker-compose-nexus-add-mqtt.yml \
+		-f docker-compose-nexus-add-security.yml \
+		up -d
+
+run-mqtt-arm-no-sec:
+	ARCH="-arm64" \
+	REDIS_ARCH="arm64v8/" \
+	docker-compose -p edgex \
+		-f docker-compose-nexus-base.yml \
+		up -d
+
+run-ui:
+	docker-compose -p edgex \
+		-f docker-compose-nexus-ui.yml \
+		up -d
+
+run-ui-arm:
+	ARCH="-arm64" \
+	docker-compose -p edgex \
+		-f docker-compose-nexus-ui.yml \
+		up -d
+
+down:
+	docker-compose  -p edgex \
+		-f docker-compose-nexus-base.yml \
+		-f docker-compose-nexus-add-mqtt.yml \
+		-f docker-compose-nexus-add-security.yml \
+		-f docker-compose-nexus-ui.yml \
+		down
+
+clean: down
+	-docker rm $$(docker ps -aq)
```

Comment:
This line is hazardous to my system as I have other containers running beside edgex's.  It is trying to remove these other containers.

Reply:
This only removes them if the container is stopped. I can look at refining it, but need to ensure that all the containers used be the stack are removed before pruning. otherwise the prune will not get everything.

Reply:
I have observed that there is no ill side effects to running docker-compose "down" on stacks that I haven't done an "up" on.  In other words, why not "docker-compose -f *.yml down"

Reply:
I can filer on the network used to just hit the edgex stack containers.

Reply:
While you're at it, this might be a good time to explicitly name the network instead of inheriting it from cwd.

Reply:
Already done. By specifying the `-p edgex` on the docker-compose command. This puts all the containers, networks, volumes, etc. under the `edgex` stack and names them with the `edgex_` prefix.

Reply:
> I have observed that there are no ill side effects to running docker-compose "down" on stacks that I haven't done an "up" on. In other words, why not "docker-compose -f *.yml down"

Nice idea, but doesn't work as the `-f` needs to be repeated with each file. Some complex scripting could create command properly, but IMO this work fine with just 4 files. More files might justify a more dynamic approach.